### ISSUE TITLE
code(method) does not give code for the method

### ISSUE THREAD

**Issue by b-stone** [Macaulay2/M2#456]
I would like to be able to run `code method` and have the code of the method displayed. For example, when running `code Ext` I get the following.

```
i13 : code Ext
stdio:13:1:(3): error: no method found for applying code to:
     argument   :  Ext (of class ScriptedFunctor)
```

I would like to see where in the code this method is defined.

**Comment by DanGrayson** [comment]
Yes, currently one has to type `code methods Ext`, but we could make that automatic.
